fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -1045,7 +823,8 @@ ${RUN} ${PipelineScripts}/OneStepResampling.sh \
        --oscout=${fMRIFolder}/${NameOffMRI}_SBRef_nonlin \
        --ojacobian=${fMRIFolder}/${JacobianOut}_MNI.${FinalfMRIResolution} \
        --fmrirefpath=${fMRIReferencePath} \
-       --fmrirefreg=${fMRIReferenceReg}
+       --fmrirefreg=${fMRIReferenceReg} \
+       --wb-resample=1 #REMOVE AFTER TESTING!!
```

Comment:
Fix this.

Reply:
This is more than cosmetic, and could interfere with qunex testing.

Reply:
Done